chart/values.yaml

```diff
@@ -1,26 +1,26 @@
 replicaCount: 1
 
 image:
-  repository: ubiviushub/microservice-template
+  repository: ubiviushub/microservice-authentication
   tag: 0.0.0
   pullPolicy: Always
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
-readinessProbe:
-  httpGet:
-    path: /products # TODO: Change for /health
-    port: 9090
+# readinessProbe:
```

Comment:
Without any path for readiness/liveness, image will always be rebooted by k8s api. Is there a reason why code is commented?

Reply:
Je vais me faire une tâche pour implémenter un path pour les readiness/liveness dans tous les microservices, pcq signup et signin retourne rien pour le moment donc j'ai toujours un warning

Reply:
That seems fair